src/utils/daemon.rs

```diff
@@ -66,29 +63,24 @@ pub fn start_daemon(context: &dyn Context, enable_registry_watcher: bool) -> Res
     // build new crates every minute
     let pool = context.pool()?;
     let build_queue = context.build_queue()?;
-    let cloned_config = config.clone();
     let rustwide_builder = RustwideBuilder::init(context)?;
     thread::Builder::new()
         .name("build queue reader".to_string())
         .spawn(move || {
-            let doc_builder =
-                DocBuilder::new(cloned_config.clone(), pool.clone(), build_queue.clone());
+            let doc_builder = DocBuilder::new(config.clone(), pool.clone(), build_queue.clone());
             queue_builder(doc_builder, rustwide_builder, build_queue).unwrap();
         })
         .unwrap();
 
-    if let Some(github_updater) = GithubUpdater::new(config, context.pool()?)? {
-        cron(
-            "github stats updater",
-            Duration::from_secs(60 * 60),
-            move || {
-                github_updater.update_all_crates()?;
-                Ok(())
-            },
-        )?;
-    } else {
-        log::warn!("GitHub stats updater not started as no token was provided");
-    }
+    let updater = context.repository_stats_updater()?;
```

Comment:
Just for my sanity - this isn't a change in behavior, it will still skip repo updates and continue if no token is provided. The only time this can return an error is when creating a pool or config fails, which shouldn't happen here because this is run right at startup.

Can you add that as a comment?

Reply:
Sure!